js/modules/k6/experimental/tracing/propagator.go

```diff
@@ -69,11 +69,11 @@ const (
 	// "0 value is valid and means “root span” (when not ignored)"
 	JaegerRootSpanID = "0"
 
-	// JaegerSampledTraceFlag is the trace-flag value for an unsampled trace.
-	JaegerSampledTraceFlag = "0"
+	// JaegerUnsampledTraceFlag is the trace-flag value for an unsampled trace.
+	JaegerUnsampledTraceFlag = "0"
 
-	// JaegerUnsampledTraceFlag is the trace-flag value for a sampled trace.
-	JaegerUnsampledTraceFlag = "1"
+	// JaegerSampledTraceFlag is the trace-flag value for a sampled trace.
+	JaegerSampledTraceFlag = "1"
```

Comment:
This is a bit strange to me. It is a boolean, isn't possible to have a single value so a single source of truth?

Reply:
This is a small simplification. The sampled/not sampled flag is specified as part of a set of flags: a bit within a byte. 

As per the specification it could have a range of values. It just so happens that we're only using the sample flag at the moment, and encode it, at the end of the chain as a string, so keeping the constant as a string is a shortcut that avoids adding conversion code.